SPECS/ruby/ruby.spec

```diff
@@ -46,31 +108,153 @@ BuildRequires:  sudo
 
 Requires:       gmp
 Requires:       openssl
-
 Provides:       %{_prefix}/local/bin/ruby
 Provides:       %{name}-devel = %{version}-%{release}
 Provides:       %{name}(release) = %{version}-%{release}
 Provides:       %{name}-libs = %{version}-%{release}
-# TODO: When moving to Ruby 3.X for Mariner 2.0 release, these gemified stdlib
-# provides should be versioned according to the gem version.
+# TODO: When changing ruby version, these gemified stdlib
+# provides should be versioned according to the ruby version.
```

Comment:
Please also include a similar comment above the `Version` tag - changing that will most likely require updates to the bundled gems' versions.